tests/by-util/test_touch.rs

```diff
@@ -34,7 +34,7 @@ fn set_file_times(at: &AtPath, path: &str, atime: FileTime, mtime: FileTime) {
 
 fn str_to_filetime(format: &str, s: &str) -> FileTime {
     let tm = chrono::NaiveDateTime::parse_from_str(s, format).unwrap();
-    FileTime::from_unix_time(tm.and_utc().timestamp(), tm.timestamp_subsec_nanos())
+    FileTime::from_unix_time(tm.and_utc().timestamp(), tm.and_utc().timestamp_subsec_nanos())
```

Comment:
rustfmt isn't happy

```suggestion
    FileTime::from_unix_time(
        tm.and_utc().timestamp(),
        tm.and_utc().timestamp_subsec_nanos(),
    )
```

Reply:
Thank you! And silly me, I thought I did ran `cargo fmt` but maybe I forgo.